### PR TITLE
Control status bar button visibility

### PR DESCRIPTION
**Pull request by drehelis**
https://github.com/metalbear-co/mirrord-vscode/issues/157

### CODE REVIEW

File: changelog.d/+157.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Add an option to control mirrord status bar button visibility.
```
Comment (aviramha) [inline changelog.d/+157.md:1]: Change filename to `157.added.md`

File: src/status.ts [hunk 2]
```diff
@@ -18,6 +18,13 @@ export class MirrordStatus {
     }
 
     draw() {
+        const showStatusBar = vscode.workspace.getConfiguration('mirrord').get('showStatusBarButton', true);
+        
+        if (!showStatusBar) {
+            this.statusBar.hide();
```
Comment (aviramha) [inline src/status.ts:24]: I wonder if you checked if `return` without `hide` suffices, since we explicitly call `show` below?

**Review by aviramha — CHANGES_REQUESTED** [review]
Thank you for the contribution - I have few remarks.

File: src/status.ts (hunk 2 above)
Comment (drehelis) [reply src/status.ts]: It does. removed it.

**Review by aviramha — APPROVED** [review]
LGTM, Thank you

**Pull request merged**